site/public/js/forum.js

```diff
@@ -788,7 +788,8 @@ function showEditPostForm(post_id, thread_id, shouldEditThread, render_markdown,
                 var thread_title = json.title;
                 var thread_lock_date =  json.lock_thread_date;
                 var thread_status = json.thread_status;
-                var expiration = json.expiration;
+                var expiration = json.expiration.replace("-", "/");
+                expiration = expiration.replace("-", "/");
```

Comment:
You can use the `g` flag on regex to match all elements in a replacement. Given that there's only two `/` characters and no more (unless we have an invalid date), should be fine to do:
```suggestion
                const expiration = json.expiration.replace(/-/g, "/");
```

Reply:
I didn't use the `g` flag since there is another `-` that shouldn't get changed. 
Here is how the date is stored in the database `2021-09-27 00:00:00-04`

Reply:
Oh okay, well this is certainly a mess, and all the more reason to move away from `Date.parse`.